src/firebase.js

```diff
@@ -0,0 +1,13 @@
+import firebase from 'firebase'
+const config = {
+  apiKey: 'AIzaSyBvk55SfH5m_8npStGRblPXNuhY--t27Y4',
+  authDomain: 'group-photos-30233.firebaseapp.com',
+  databaseURL: 'https://group-photos-30233.firebaseio.com',
+  projectId: 'group-photos-30233',
+  storageBucket: 'group-photos-30233.appspot.com',
+  messagingSenderId: '133784701267'
```

Comment:
So this stuff you want to *never* commit to source code. You are basically giving away all the permissions to the application. This file should go into your `.gitignore` file, and in your `README.md` file, you should provide instructions for how to set this file up.

Reply:
added to .gitingore.  needs to double check that file path worked and it's actually removed from GH